backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Entities/GasStationEntity.java

```diff
@@ -0,0 +1,43 @@
+package bo.edu.usfa.gasolina.habragasolina.Entities;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "gas_stations")
+public class GasStationEntity {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id_gas_station;
```

Comment:
this line is bad: 
it should be like this:   
`private Integer id_gas_station`